helpers/cuentasContablesHelper/cuentasContables.helper.go

```diff
@@ -265,5 +265,27 @@ func AsientoContable(totales map[int]float64, tipomvto string, descripcionMovto
 			return nil, outputError
 		}
 	}
+
+	apiMvtoContables := "http://" + beego.AppConfig.String("movimientosContablesmidService") + "transaccion_movimientos/transaccion_movimientos/"
+	if err := request.SendJson(apiMvtoContables, "POST", &res, &transaccion); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "AsientoContable -if err := request.SendJson(apiMvtoContables, \"POST\", &res, &transaccion);",
+			"err":     err,
+			"status":  "502",
+		}
+	} else {
+		logs.Info("Transaccion contable")
+		logs.Info("******** correcto *********")
```

Comment:
Comentar
```suggestion
		// logs.Info("Transaccion contable")
		// logs.Info("******** correcto *********")
```

o Eliminar
```suggestion
```